src/scss/4_state/layout/cards.scss

```diff
@@ -128,51 +128,107 @@
   }
 
   .card {
-    &__image {
 
-      img {
-        width: 100%;
-        max-width: 300px;
-        margin: 0;
-      }
+    &__link {
+
+      // Wraps the entire card.
+      display: flex;
+      flex-flow: column nowrap;
+      align-items: flex-start;
+      width: 100%;
+      height: 100%;
+      text-decoration: none !important;
```

Comment:
Is there a way to introduce this override without using `!important`?

Reply:
Yep good spot - that !important rule can be removed - not needed at all